src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/Staff.kt

```diff
@@ -1,16 +1,21 @@
 package uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa
 
+import org.hibernate.Hibernate
 import javax.persistence.Column
 import javax.persistence.Entity
+import javax.persistence.GeneratedValue
 import javax.persistence.Id
+import javax.persistence.SequenceGenerator
 import javax.persistence.Table
 
 @Entity
 @Table(name = "STAFF_MEMBERS")
 data class Staff(
   @Id
+  @SequenceGenerator(name = "STAFF_ID", sequenceName = "STAFF_ID", allocationSize = 1)
+  @GeneratedValue(generator = "STAFF_ID")
   @Column(name = "STAFF_ID", nullable = false)
-  val staffId: Long,
+  var staffId: Long = 0,
 
```

Comment:
0 (or negative) means Hibernate will generate a primary key from sequence, but nicely means it is not nullable in kotlin